src/lib/skilling/skills/herblore/mixables/barbMixes.ts

```diff
@@ -0,0 +1,390 @@
+		id: itemID('Antidote+ mix(2)'),
+		level: 74,
+		xp: 52,
+		inputItems: new Bank({
+			'Antidote+(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	},
+	{
+		name: 'Antifire mix(2)',
+		aliases: ['antifire mix(2)', 'antifire mix', 'antifire mix caviar'],
+		id: itemID('Antifire mix(2)'),
+		level: 75,
+		xp: 53,
+		inputItems: new Bank({
+			'Antifire potion(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	},
+	{
+		name: 'Ranging mix(2)',
+		aliases: ['ranging mix(2)', 'ranging mix', 'ranging mix caviar'],
+		id: itemID('Ranging mix(2)'),
+		level: 80,
+		xp: 54,
+		inputItems: new Bank({
+			'Ranging potion(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	},
+	{
+		name: 'Magic mix(2)',
+		aliases: ['Magic mix(2)', 'Magic mix', 'Magic mix caviar'],
+		id: itemID('Magic mix(2)'),
+		level: 83,
+		xp: 57,
+		inputItems: new Bank({
+			'Magic potion(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	},
+	{
+		name: 'Zamorak mix(2)',
+		aliases: ['Zamorak mix(2)', 'Zamorak mix', 'Zamorak mix caviar'],
+		id: itemID('Zamorak mix(2)'),
+		level: 85,
+		xp: 58,
+		inputItems: new Bank({
+			'Zamorak brew(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	},
+	{
+		name: 'Stamina mix(2)',
+		aliases: ['Stamina mix(2)', 'Stamina mix', 'Stamina mix caviar'],
+		id: itemID('Stamina mix(2)'),
+		level: 86,
+		xp: 60,
+		inputItems: new Bank({
+			'Stamina potion(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	},
+	{
+		name: 'Extended antifire mix(2)',
+		aliases: ['extended antifire mix(2)', 'extended antifire mix', 'extended antifire mix caviar'],
+		id: itemID('Extended antifire mix(2)'),
+		level: 91,
+		xp: 61,
+		inputItems: new Bank({
+			'Extended antifire(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	},
+	{
+		name: 'Ancient mix(2)',
+		aliases: ['ancient mix(2)', 'Ancient mix', 'Ancient mix caviar'],
+		id: itemID('Ancient mix(2)'),
+		level: 92,
+		xp: 63,
+		inputItems: new Bank({
+			'Ancient brew(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	},
+	{
+		name: 'Super antifire mix(2)',
+		aliases: ['super antifire mix(2)', 'super antifire mix', 'super antifire mix caviar'],
+		id: itemID('Super antifire mix(2)'),
+		level: 98,
+		xp: 70,
+		inputItems: new Bank({
+			'Super antifire potion(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	},
+	{
+		name: 'Extended super antifire mix(2)',
+		aliases: [
+			'extended super antifire mix(2)',
+			'extended super antifire mix',
+			'extended super antifire mix caviar'
+		],
+		id: itemID('Extended super antifire mix(2)'),
+		level: 99,
+		xp: 78,
+		inputItems: new Bank({
+			'Extended super antifire(2)': 1,
+			Caviar: 1
+		}),
+		tickRate: 1,
+		bankTimePerPotion: 0.088
+	}
+];
+
+export default barbMixes;
```

Comment:
we dont use export defaults anywhere. you were just copying how the other imports do that, so you were doing a smart thing, but anything using export default was written a while ago, and we want to move away from that in the future.